pkgs/servers/home-assistant/appdaemon.nix

```diff
@@ -6,6 +6,14 @@
 let
   python = python3.override {
     packageOverrides = self: super: {
+      astral = super.astral.overridePythonAttrs (oldAttrs: rec {
```

Comment:
Can you add a link to an upstream issue?

Reply:
There is no upstream issue. The derivation introduces the regression by unpinning the astral package version. See the commit message for detailed information. Here's a traceback of the symptom:

```
Mar 17 00:38:00 nixos appdaemon[9134]: Traceback (most recent call last):
Mar 17 00:38:00 nixos appdaemon[9134]:   File "/nix/store/la71x6y3npq1fzrvkv2f7kb481c2yls4-appdaemon-4.0.5/lib/python3.8/site-packages/appdaemon/__main__.py", line 117,>
Mar 17 00:38:00 nixos appdaemon[9134]:     self.AD = ad.AppDaemon(self.logging, loop, **appdaemon)
Mar 17 00:38:00 nixos appdaemon[9134]:   File "/nix/store/la71x6y3npq1fzrvkv2f7kb481c2yls4-appdaemon-4.0.5/lib/python3.8/site-packages/appdaemon/appdaemon.py", line 187>
Mar 17 00:38:00 nixos appdaemon[9134]:     self.sched = scheduler.Scheduler(self)
Mar 17 00:38:00 nixos appdaemon[9134]:   File "/nix/store/la71x6y3npq1fzrvkv2f7kb481c2yls4-appdaemon-4.0.5/lib/python3.8/site-packages/appdaemon/scheduler.py", line 62,>
Mar 17 00:38:00 nixos appdaemon[9134]:     self.init_sun()
Mar 17 00:38:00 nixos appdaemon[9134]:   File "/nix/store/la71x6y3npq1fzrvkv2f7kb481c2yls4-appdaemon-4.0.5/lib/python3.8/site-packages/appdaemon/scheduler.py", line 220>
Mar 17 00:38:00 nixos appdaemon[9134]:     self.location = astral.Location(("", "", latitude, longitude, self.AD.tz.zone, elevation))
Mar 17 00:38:00 nixos appdaemon[9134]: AttributeError: module 'astral' has no attribute 'Location'
```